dohq_artifactory/admin.py

```diff
@@ -741,6 +743,12 @@ def _create_json(self):
             "archiveBrowsingEnabled": self.archiveBrowsingEnabled,
             "yumRootDepth": 0,
         }
+        """
+        Docker V2 API specific fields
+        """
+        if Repository.V2 == self.dockerApiVersion:
```

Comment:
would be more readable to have 
```suggestion
        if self.dockerApiVersion == Repository.V2:
```